### DISCUSSION TITLE
Goals section not available

### DISCUSSION THREAD

**Discussion by vishnurm19** [ananthakumaran/paisa#98] [Q&A]
In the configuration tab, I'm only able to see the Retirement section. The Goals section is not visible, neither is the option to add a goal.
Would appreciate any help. Images attached.

![2023-11-13_20-56](https://github.com/ananthakumaran/paisa/assets/150701983/f01d6681-ad13-4963-a0e6-8dfa844f0af7)
![2023-11-13_20-57](https://github.com/ananthakumaran/paisa/assets/150701983/b091a075-f514-4782-a57e-876bbb6d1c2b)

**Comment by ananthakumaran** [comment]
@vishnurm19  can you check the paisa version (More > About). Goals are available only from 5.7 onwards.

**Reply by vishnurm19** [reply]
Thank you for your reply! Mine was 5.6, and I didn't know about the new release. Solved.
